src/blocks/Header/MobileHeader.js

```diff
@@ -71,8 +100,12 @@ const MobileHeader = ({ headerProps }) => {
   return (
     <Grid as="nav">
       <Logo />
-      <DialogDisclosure {...dialog} style={{ color: 'white', height: 24 }}>
-        {dialog.visible ? <Cross height={24} /> : <Hamburger height={24} />}
+      <DialogDisclosure {...dialog} style={{ color: 'white', height: '24px' }}>
+        <MenuBtnWrapper clicked={dialog.visible}>
+          <MenuBtnBar />
+          <MenuBtnBar />
+          <MenuBtnBar />
+        </MenuBtnWrapper>
```

Comment:
Could you create a `Hamburger` component out of it and add it in an apart file?

Reply:
Sure. working on it. I will create file in same folder regarding it.

Reply:
I prefered you add it in the `components` folder.